web/src/app/administration/components/workbasket-overview/workbasket-overview.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { Select, Store } from '@ngxs/store';
+import { Observable, Subject } from 'rxjs';
+import { ActivatedRoute } from '@angular/router';
+import { takeUntil } from 'rxjs/operators';
+import { WorkbasketSelectors } from '../../../shared/store/workbasket-store/workbasket.selectors';
+
+import { CreateWorkbasket,
+  SelectWorkbasket,
+  SetActiveAction } from '../../../shared/store/workbasket-store/workbasket.actions';
+
+@Component({
+  selector: 'app-workbasket-overview',
+  templateUrl: './workbasket-overview.component.html',
+  styleUrls: ['./workbasket-overview.component.scss']
+})
+export class WorkbasketOverviewComponent implements OnInit {
+  showDetail = false;
+  @Select(WorkbasketSelectors.selectedWorkbasketAndAction) selectedWorkbasketAndAction$: Observable<any>;
+  destroy$ = new Subject<void>();
+  routerParams: any;
+
+  constructor(
+    private route: ActivatedRoute,
+    private store: Store
+  ) {
+
+  }
+
+  ngOnInit() {
+    if (this.route.firstChild) {
+      this.route.firstChild.params
+        .pipe(takeUntil(this.destroy$))
```

Comment:
a take(1) would suffice in this case, as far as i can see, because we do not activate Route again.
This is just a preference and should not change anything

Reply:
when the user changes the route manually via browser address, it needs a subscription to detect these changes. Without it you need to enter twice to force refresh the site.